gulpfile.js

```diff
@@ -143,6 +143,8 @@ function javascript() {
 function images() {
   return gulp
     .src('src/assets/img/**/*')
+    .pipe(webp())
+    .pipe(gulp.dest(PATHS.dist + '/assets/img'))
```

Comment:
In my local tests, this only generates the webp files, not the minified png and jpg files. If I switch the order, only png and jpg are created but not webp

Reply:
creating a separate function and adding it separately to the series seems to work, though

Reply:
Good point, i will provide the fix